### DISCUSSION TITLE
URL Rewriting

### DISCUSSION THREAD

**Discussion by brnsgit** [sveltejs/kit#8921] [Q&A]
Hello,

I'm using SvelteKit on CloudFlare Pages and I'm interested in rewriting all the URLs of the images hosted on the domain, to another domain. To be more clear, I want something like this:
source: '/images/:slug',
destination: 'https://anotherdomain.com/images/:slug*'
...and this rule to apply on the entire app. 

How can I manage to do that?

**Comment by ivoilic** [comment]
You can do this with the [`handle` server hook](https://kit.svelte.dev/docs/hooks#server-hooks-handle).
Within the hook check the path and either use a [redirect](https://kit.svelte.dev/docs/load#redirects) or proxy the request by running a fetch request on the server and returning the result.
